config/foreman.migrations/20160516143923_remove_ovirt_provision.rb

```diff
@@ -1,3 +0,0 @@
-# Remove unmaintained ovirt_provision plugin
```

Comment:
It's fine to remove it IIRC, but a new migration should be added to put the config/answer entries back.